File: Domain/Schedule.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Domain
{
    public class Schedule
    {

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public int MondayConsultations { get; set; }

        public int TuesdayConsultations { get; set; }

        public int WednesdayConsultations { get; set; }

        public int ThursdayConsultations { get; set; }

        public int FridayConsultations { get; set; }


    }
}

File: bettercalm-frontend/src/app/models/schedule.ts
import { Injectable } from '@angular/core';
import { Adapter } from './adapter';

export class Schedule {
    constructor(
        public id: number,
        public mon: number,
        public tue: number,
        public wed: number,
        public thu: number,
        public fri: number
    ) { }
}

@Injectable({
    providedIn: 'root'
})
export class ScheduleAdapter implements Adapter<Schedule> {
    adapt(item: any): Schedule {
        return new Schedule(
            item.id,
            item.mondayConsultations,
            item.tuesdayConsultations,
            item.wednesdayConsultations,
            item.thursdayConsultations,
            item.fridayConsultations
        )
    }
}
